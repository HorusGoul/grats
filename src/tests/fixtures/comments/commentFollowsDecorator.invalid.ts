@ObjectType()
/** @gqlType */
export default class Composer {
  @Field()
  /** @gqlField */
  url(): string {
    return `/composer/`;
  }
}
