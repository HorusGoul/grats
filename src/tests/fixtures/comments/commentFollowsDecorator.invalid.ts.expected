-----------------
INPUT
----------------- 
@ObjectType()
/** @gqlType */
export default class Composer {
  @Field()
  /** @gqlField */
  url(): string {
    return `/composer/`;
  }
}

-----------------
OUTPUT
-----------------
src/tests/fixtures/comments/commentFollowsDecorator.invalid.ts:2:5 - error: Unexpected Grats tag in detached docblock. Grats was unable to determine which TypeScript declaration this docblock is associated with. Moving the docblock to a position with is unambiguously "above" the relevant declaration may help.

2 /** @gqlType */
      ~~~~~~~~
src/tests/fixtures/comments/commentFollowsDecorator.invalid.ts:5:7 - error: Unexpected Grats tag in detached docblock. Grats was unable to determine which TypeScript declaration this docblock is associated with. Moving the docblock to a position with is unambiguously "above" the relevant declaration may help.

5   /** @gqlField */
        ~~~~~~~~~
